docs/developer_guide.md

```diff
@@ -82,14 +82,37 @@ root of this repo.
 
 ```bash
 # In repo root directory
-echo mixer_api_key=<YOUR API KEY>
+export mixer_api_key=<YOUR API KEY>
 go run cmd/main.go \
     --use_sqlite=true \
     --use_bigquery=false \
     --use_base_bigtable=false \
     --use_branch_bigtable=false \
     --use_maps_api=false \
-    --sqlite_path=$PWD/data \
+    --sql_data_path=$PWD/data \
```

Comment:
Wondering if this should be called something like `csv_data_path`?

Reply:
In sqlite case, this is also the folder that contains the db files. I think sql_xx conveys the message that this is a folder for sql data?